docs/changelog.rst

```diff
@@ -8,6 +8,10 @@ CHANGELOG
 **Improvements**
 
 - Sensitivity: Add missing attachments list to sensitive areas API
+- Sensitivity: Add ``openair`` export format for aerial sensitive areas. Usage example:
```

Comment:
I would rather include exampls in the related issue (https://github.com/GeotrekCE/Geotrek-admin/issues/2372) and add a reference to this one in the changelog : 

- Sensitivity: Add ``openair`` export format for aerial sensitive areas (#2372)

Reply:
Ok, fixed